docs/core/deploying/index.md

```diff
@@ -185,17 +185,21 @@ Deploying a self-contained app with no third-party dependencies involves creatin
     }
     ```
 
-3. Open the `project.json` file and find the `dependencies` section, which should appear as follows:
+3. Open the `project.json` file and replace the `frameworks` section with the following:
 
     ```json
-    "dependencies": {
-       "Microsoft.NETCore.App": {
-         "type": "platform",
-         "version": "1.0.0"
-       }
-    },
+    "frameworks": {
+      "netstandard1.6": { }
+    }
     ```
-   Replace it with the following:
+
+This does two things:
+
+    * It indicates that, instead of using the entire `netcoreapp1.0` framework, which includes .NET Core CLR, the .NET Core Library, and a number of other system components, our app uses only the .NET Standard Library.
```

Comment:
Try 3 spaces here instead of four.  This renders as code when you do four spaces before the asterisk.